src/bp-groups/bp-groups-functions.php

```diff
@@ -3619,7 +3619,7 @@ function bp_register_active_group_types() {
 						'name'          => $name,
 						'singular_name' => $singular_name,
 					),
-					'has_directory'         => strtolower( $name ),
+					'has_directory'         => strtolower( $key ),
```

Comment:
@kishanjasani I think we no need to pass any value we just pass `true` and default It'll take group type as directory slug.

Reply:
Thanks for the review. Added a changes